src/cockpit/beiboot.py

```diff
@@ -181,9 +196,20 @@ def __init__(self, router: Router, destination: str, args: argparse.Namespace):
         super().__init__(router)
 
     async def do_connect_transport(self) -> None:
+        # do we have user/password (Basic auth) from the login page?
+        auth = await self.router.request_authorization("*")
```

Comment:
@allisonkarlitskaya IIRC you were worried that this PR changes the Client. It's not supposed to, as that will just "fail" (i.e. give empty-ish reply `":"`) on the client, as the login page doesn't *have* user and password fields. This only applies to using cockpit-beiboot as cockpit-ssh replacement on the "normal" login page.